snapcraft.yaml

```diff
@@ -54,6 +54,8 @@ parts:
       - authz_core
       - unixd
       - alias
+      - proxy
+      - proxy_fcgi
```

Comment:
Do these actually contribute to better performance, or do they relate to another issue (and thus should be in a different PR)?